2020-an-environment-for-sustainable-research-software-in-germany-and-beyond/manuscript.md

```diff
@@ -0,0 +1,2020 @@
+domains (highlighted in bold) and additional desirable, soft criteria
+that can be implemented to different degrees depending on the use case
+and domain-specific software development requirements. The soft criteria
+should be evaluated in a structured way by the reviewers with a specific
+response for each section rather than one running text. The fact that
+most of these criteria will be considered in any software management
+plan (SMP, compare section   on p. ) (SSI 2019b) highlights its
+importance for sustainable research software.
+
+### Usage and Impact
+
+Requirements qualifying software for sustained funding are (1) its **use
+beyond a single research group**, (2) the scientific relevance and
+validity of the software documented in **at least one peer-reviewed
+scientific publication**. Ideally a paper also describes the scope,
+performance and design of the software. (3) The use of the software in
+publications is a measure of impact but quantitative assessment brings
+about additional challenges (Li, Chen, and Yan 2019). Therefore, other,
+potentially domain-specific, impact measures, such as influence on
+policy and practice as well as use in other software and products should
+be considered as well to evaluate relevance for academia and society.
+Considerable attendance at training and networking events can be
+considered as a proof of use as well. (4) A **market analysis** needs to
+show that the software is important to a user base of relevant size and
+either unique or one of the main players in a field with several
+existing solutions. Geographical or political aspects can be considered
+as well, e.g. to support the maintenance of a European solution. A
+convergence process of (parts of) a research community towards a
+specific software stack, i.e. documented transition of several research
+groups to a common software, would be a strong indicator of impact.
+(5) As community benefits are a central goal of sustained software
+funding, outreach and **appropriate training material** for new users of
+the software are essential.
+
+### Software Quality
+
+As mandatory criteria of software quality that have to be fulfilled, we
+consider (6) the **public availability of the source code**, with
+(7) **version control** (ideally on a public platform).
+(8) **Documentation** of the software needs to be publicly available
+comprising both user documentation (requirements, installation, getting
+started, user manual) and developer documentation (ideally within the
+code, e.g. using Doxygen) (Lee 2018). (9) The **license** under which
+the software is distributed must be defined and ensure public code
+availability and free use, at least for academic purposes.
+(10) **Dependencies** on libraries and technologies must be defined.
+
+We acknowledge that some additional criteria have to be evaluated under
+consideration of the research domain. These comprise (11) the
+availability of examples (comprising input data and reference results),
```

Comment:
```suggestion
availability of **examples** (comprising input data and reference results),
```

Reply:
the non-bold points were meant as not strictly necessary requirements (see above, "This set comprises
 mandatory, hard criteria that we think have to be fulfilled across domains (highlighted in bold) and additional desirable, soft criteria that can be implemented to different degrees depending on the use case
 and domain-specific software development requirements.")